superset/db_engine_specs/oracle.py

```diff
@@ -15,7 +15,12 @@
 # specific language governing permissions and limitations
 # under the License.
 from datetime import datetime
-from typing import Optional
+from typing import (
+    Any,
+    List,
+    Optional,
+    Tuple
+)
```

Comment:
This is what `isort` would do:
```suggestion
from typing import Any, List, Optional, Tuple
```
Check https://github.com/apache/incubator-superset/blob/master/CONTRIBUTING.md for details on how to install pre-commit hooks to do this automatically:

```bash
pip3 install -r requirements/integration.txt
pre-commit install
```

Reply:
ok